pkgs/top-level/all-packages.nix

```diff
@@ -25791,6 +25791,14 @@ in
 
   octopus = callPackage ../applications/science/chemistry/octopus { };
 
+  openlp = libsForQt514.callPackage ../applications/misc/openlp { };
```

Comment:
Can Qt5 be used here?

Reply:
I will change it.

Reply:
Resolved in 660a3abb0850535093bff14b1c8bca56a6ef8cb1.